spirv-builder/src/test/mod.rs

```diff
@@ -1,10 +1,22 @@
 mod basic;
 
 use lazy_static::lazy_static;
+use rustc_codegen_spirv::rspirv;
 use std::error::Error;
 use std::path::{Path, PathBuf};
 use std::sync::Mutex;
 
+// https://github.com/colin-kiegel/rust-pretty-assertions/issues/24
+#[derive(PartialEq, Eq)]
+#[doc(hidden)]
```

Comment:
Is there a reason to hide `PrettyString`?
```suggestion
```

Reply:
I copypasted it from the linked issue without thinking, shrug

Reply:
Should it be `pub(crate)` anyway?